core/foundation/inc/ROOT/StringUtils.hxx

```diff
@@ -40,6 +40,8 @@ std::string Join(const std::string &sep, StringCollection_t &&strings)
                           [&sep](auto const &a, auto const &b) { return a + sep + b; });
 }
 
+std::string Round(const double value, const double error, const unsigned short cutoff = 1, const std::string delim = "#pm");
```

Comment:
The last param should be a `std::string_view`, or at least a `const std::string &` (but string_view is better as it doesn't force an allocation of what will 90% be a constant string literal)